domain/src/test/java/org/oppia/android/domain/classify/InteractionObjectBuilder.kt

```diff
@@ -0,0 +1,88 @@
+package org.oppia.android.domain.classify
+
+import org.oppia.android.app.model.Fraction
+import org.oppia.android.app.model.InteractionObject
+import org.oppia.android.app.model.ListOfSetsOfHtmlStrings
+import org.oppia.android.app.model.StringList
+import javax.inject.Inject
+
+/** Helper class which can provide with the [InteractionObject] of a particular ObjectTypeCase. */
```

Comment:
/** Test helper class which can provide with the [InteractionObject] of a particular ObjectTypeCase. */

In someway we should mention `test` word in this comment.

Reply:
/** Helper class for test cases which can provide with the [InteractionObject]
 * of a particular ObjectTypeCase.
 */

This sounds good to me.